docs/docs/thinking-in-react.md

```diff
@@ -114,7 +114,7 @@ Remember: React is all about one-way data flow down the component hierarchy. It
 For each piece of state in your application:
 
   * Identify every component that renders something based on that state.
-  * Find a common owner component (a single component above all the components that need the state in the hierarchy).
+  * Find a common owner component (a single component above all the components that needs the state in the hierarchy).
```

Comment:
I understand your point but the grammar isn't quite there I think. However, if you insist on the singular form, maybe it would be better to say, 
__Find a common owner component, i.e. a single component above all the components that need "state", in the hierarchy).__